diff -ur openldap-2.4.25/configure openldap-2.4.25.new/configure
--- openldap-2.4.25/configure	2011-03-24 02:52:17.000000000 +0100
+++ openldap-2.4.25.new/configure	2011-10-12 22:46:10.161531481 +0200
@@ -2026,6 +2026,15 @@
      ) | sed "s/^/$as_me: WARNING:     /" >&2
     ;;
 esac
+echo
+echo Minitage CFLAGS/LDFLAGS switching
+echo " $CFLAGS -> $MINITAGE_CFLAGS"
+echo " $LDFLAGS -> $MINITAGE_LDFLAGS"
+CFLAGS="$MINITAGE_CFLAGS"
+CPPFLAGS="$MINITAGE_CPPFLAGS"
+LDFLAGS="$MINITAGE_LDFLAGS"
+export CFLAGS LDFLAGS CPPFLAGS
+echo 
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for $2" >&5
 $as_echo_n "checking for $2... " >&6; }
 if { as_var=$3; eval "test \"\${$as_var+set}\" = set"; }; then :
@@ -21283,6 +21292,8 @@
 		minor != DB_VERSION_MINOR ||
 		patch != DB_VERSION_PATCH )
 	{
+        printf("\d.\d.\d\n", major,minor,patch);
+        printf("\d.\d.\d\n", DB_VERSION_MAJOR, DB_VERSION_MINOR, DB_VERSION_PATCH); 
 		printf("Berkeley DB version mismatch\n"
 			"\theader: %s\n\tlibrary: %s\n",
 			DB_VERSION_STRING, version);
